.ci/script_catkin.sh

```diff
@@ -20,7 +20,7 @@ fi
 # Build aikidopy and pytest
 if [ $BUILD_AIKIDOPY = ON ]; then
   ./scripts/internal-run.sh catkin build --no-status --no-deps -p 1 -i --cmake-args -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DTREAT_WARNINGS_AS_ERRORS=OFF -DCODECOV=OFF --make-args aikidopy -- aikido
-  ./scripts/internal-run.sh make -C build/aikido pytest
+  $SUDO ./scripts/internal-run.sh make -C build/aikido pytest
```

Comment:
What's the reason for this change? This may be dumb, seems unrelated to the interface change.

Reply:
CI fails to run without this addition, since the installation step requires sudo.